app/src/main/res/layout/seek_bar_progress.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="@dimen/default_icon_size"
+        android:layout_height="@dimen/default_icon_size"
+        android:orientation="vertical"
+        tools:parentTag="android.widget.RelativeLayout"
+        >
+
+    <ProgressBar
+            android:id="@+id/progress"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            />
+
+    <ImageView
```

Comment:
`tools:src="@tools:sample/backgrounds/scenic` or avatar can be used for better visibility for devs